src/components/DownloadMenu/DownloadMenu.js

```diff
@@ -29,6 +29,7 @@ const DownloadMenu = ({ download, hoverable }) => {
                 onClick={() =>
                     download(DOWNLOAD_TYPE_TABLE, FILE_FORMAT_HTML_CSS)
                 }
+                className="push-analytics-download-as-html-css-menu-item"
```

Comment:
@martinkrulltott suggested this prefix and I agreed it would be good to add. The prefix is mainly there to ensure that the reader of the code gets a clear clue that these class-names are there for push-analytics to work and should not be removed. Without the prefix, the link might be unclear.